src/uct/sm/mm/base/mm_md.h

```diff
@@ -189,6 +189,46 @@ typedef struct uct_mm_component {
     UCT_COMPONENT_REGISTER(&(_var).super); \
 
 
+/*
+ * Memory-mapper component initializers for MM registration routines
+ *
+ * @param _var          Variable for MM component.
+ * @param _name         String which is the component name.
+ * @param _md_ops       Mapper operations, of type uct_mm_mapper_ops_t.
+ * @param _rkey_unpack  Remote key unpack function.
+ * @param _rkey_release Remote key release function.
+ * @param _cfg_prefix   Prefix for configuration environment vars.
+ */
+#define UCT_MM_COMPONENT_REGISTER_DEF(_var, _name, _md_ops, _rkey_unpack, \
```

Comment:
can we replace UCT_MM_COMPONENT_DEFINE?
keep name as UCT_MM_COMPONENT_DEFINE, but change the macro body

Reply:
same here: since we refactor all MM, can just replace old MM macro

Reply:
> same here: since we refactor all MM, can just replace old MM macro

yep, renamed